### ISSUE TITLE
Add descriptions to sets

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#2503]
All the good work of TheOrgg led me to wonder if we should add a description to a Button Set.

At first they would all read "No Description", but as time goes on this would be a place to record the history of the set, the person who created it, etc.

It might make #2498 not needed since the origin of the set (JourneyQuest) could go in its description.

**Comment by TheOrgg** [comment]
Would this go before the list of abilities on each set?  This actually sounds like a good idea.  I'll still use the ( )'s in the wiki as search on the wiki primarily keys off of them, but it would work well on buttonweavers.

So it'd be like "The Order of Dolls is a team of Supermodel Assassins," "Zombie Orphaus Entertainment brings the JourneyQuest set," "Home of the creations of Phil and Kaja Foglio," "Metamorphers are from the short-lived ICQ messenger edition of Button Men," etc?

Or were you thinking a much more detailed description?

**Comment by dwvanstone** [comment]
We could use it for anything. I was imaging a very quick description, like

"The ___ set was created by _____ in 2004 as part of a _________. This was the first introduction of the skill ______ to the ButtonMen world."

> Would this go before the list of abilities on each set?

There is no list of abilities on the set page. See, for example:

http://buttonweavers.com/ui/buttons.html?set=Blademasters

I would imagine the description would appear either above the buttons or below the buttons (above "Return to All Button Sets". I wouldn't want it to appear on the individual button's pages.

**Comment by irilyth** [comment]
There's also http://www.buttonweavers.com/ui/buttons.html , which may be what TheOrrg was thinking of. My inclination would not be put the description on that page, but just on the set page that dwvanstone mentioned, but I don't feel strongly about it.

**Comment by dwvanstone** [comment]
Ah - thanks! I hadn't thought of that. No, I think the details of a set belongs on that set's page.

**Comment by TheOrgg** [comment]
Yeah, the list of sets is what I was thinking of.  A super-short blurb before the skills contained in a set might make someone more inclined to click on the set (much like the list of abilities does).  A former Dragon Dice player might click on the SFR when he sees 'Dragon Dice promotional set" under the label SFR.

**Comment by irilyth** [comment]
If we wanted to do that (which seems plausible to me, FWIW), we could have two fields, "Summary" and "Description" or something.
